lib/logflare/backends/adaptor/postgres_adaptor.ex

```diff
@@ -114,12 +119,32 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptor do
 
     rows =
       for row <- result.rows do
-        result.columns |> Enum.zip(row) |> Map.new()
+        result.columns
+        |> Enum.zip(row)
+        |> Map.new()
+        |> nested_map_update()
       end
 
     {:ok, rows}
   end
 
+  defp nested_map_update(value) when is_struct(value), do: value
+
+  defp nested_map_update(value) when is_map(value),
+    do: Enum.reduce(value, %{}, &nested_map_update/2)
+
+  defp nested_map_update(value) when is_list(value), do: Enum.map(value, &nested_map_update/1)
+
+  defp nested_map_update(value), do: value
+
+  defp nested_map_update({"metadata", value}, acc) do
```

Comment:
This isn't specific to the metadata key, it applies to alll key paths

Reply:
by doing that I get the render error
<img width="1015" alt="Screenshot 2023-08-07 at 12 31 09" src="https://github.com/Logflare/logflare/assets/1697301/639bea8f-8b01-4bd5-a91d-63898fc31029">